packages/flutter/test/material/text_selection_test.dart

```diff
@@ -489,6 +489,7 @@ void main() {
               child: Material(
                 child: TextField(
                   controller: controller,
+                  maxLines: null,
```

Comment:
?

Reply:
I can't remember why I did that, I must have been debugging something.  I'll remove it, it passes without it still.